tests/cloudformation/checks/resource/aws/test_BedrockAgentEncrypted.py

```diff
@@ -0,0 +1,45 @@
+import os
+import unittest
+
+from checkov.cloudformation.checks.resource.aws.BedrockAgentEncrypted import check
+from checkov.cloudformation.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+
+class TestBedrockAgentEncrypted(unittest.TestCase):
+
```

Comment:
Maybe removing this blank line (between the class and the func) will solve the weird cfn-lint issue

Reply:
It didn't help, the issue is that `cfn-lint` should run only over yml and json files, it fails on every python file I checked locally